### ISSUE TITLE
can't find tooltip

### ISSUE THREAD

**Issue by leileizhu** [TradingPal/react-native-highcharts#21]
on the debug version,when i touch the point of line,it show tooltip, but on the release version,it can't show the tooltip when i touch the point of line, did i forgot something to add ?or what i should watch out,Do I need purchase any license for using it ?

**Comment by donaldmorton** [comment]
@leileizhu No, you don't need to purchase any license for unlocking any highcharts feature see #20 for more info on this. this has to be something else, can you provide some code maybe your config json

**Comment by leileizhu** [comment]
@donaldmorton[Object,, Object, Object, Object, Object, Object, Object, Object, Object, Object…] the Object construct is {name:"Val0" sid:"00011013" unit:"Kpa" updatime:"2017-04-14 00:01" value:"189.1"}, i follow the example to write the code......<ChartView style={{height:300}} config={conf}></ChartView>,after i have package the project, it can't show the tooltip....

**Comment by jonrh** [comment]
@leileizhu Could you please tell us what version you are using? Your description seems similar to issue #4 that affected older versions of this library. Tooltips worked in debug/simulator mode but not in release modes. The issue was that functions in the Highcharts configuration object were not being translated correctly. This should no longer be the case in the latest version.

**Comment by leileizhu** [comment]
@jonrh 1.0.1

**Comment by jonrh** [comment]
@leileizhu Thanks! In that case we have a legitimate new bug. Could you please create a repository with code that replicates/demonstrate the behaviour that you have? Something similar to [this](https://github.com/jonrh/react-native-highcharts-demo). A lot of us are running *react-native-highcharts* already in production with no problems so it's important we try to figure out what is going wrong because chances are it's happening to other people too, so thanks for letting us know!

**Comment by ecsreejith** [comment]
The tooltip issue in real devices is with the functions Highcharts.dateFormat('%Y-%m-%d %H:%M:%S', this.x),Highcharts.numberFormat(this.y, 2).

Instead of using the Highcharts functions to converts number and date to string,  use the javascript function toString(). It will work in real device also
